airflow/www/static/js/api/useSetDagRunNotes.ts

```diff
@@ -0,0 +1,61 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import axios from 'axios';
+import { useMutation, useQueryClient } from 'react-query';
+import { getMetaValue } from 'src/utils';
+import useErrorToast from '../utils/useErrorToast';
+import type { GridData } from './useGridData';
+import { emptyGridData } from './useGridData';
+
+const setDagRunNotesURI = getMetaValue('set_dag_run_note');
+
+export default function useSetDagRunNotes(
+  dagId: string,
+  runId: string,
+  newNotesValue: string,
+) {
+  const queryClient = useQueryClient();
+  const errorToast = useErrorToast();
+  const setDagRunNotes = setDagRunNotesURI.replace('_DAG_RUN_ID_', runId);
+
+  const updateGridData = (oldValue: GridData | undefined) => {
```

Comment:
```suggestion
  const updateGridData = (oldGridData: GridData | undefined) => {
```
Let's have a more descriptive name?

Reply:
When I changed the name, I got a syntax error. So I am not able to change it.

Reply:
I figured it out. This should be fixed now.